docs/en/setup/Plugins.md

```diff
@@ -14,6 +14,7 @@ or a limitation of SkyWalking auto-instrumentation (welcome to contribute!)
 Library | Python Version - Lib Version | Plugin Name
 | :--- | :--- | :--- |
 | [aiohttp](https://docs.aiohttp.org) | Python >=3.10 - NOT SUPPORTED YET; Python >=3.6 - ['3.7.4'];  | `sw_aiohttp` |
+| [bottle](http://bottlepy.org/docs/dev/) | Python >=0.12 - ['*'];  | `sw_bottle` |
```

Comment:
see here you can tell the generated doc isn't correct.

Reply:
got it.

Reply:
Is there any other error reported in Actions? I want to eliminate as many mistakes as possible on each commit. Thanks!

Reply:
It is better you could run the tests locally. Then you could catch all errors.

Reply:
> It is better you could run the tests locally. Then you could catch all errors.

Yea, I make it now.
Here is the problem I encountered.
```
docker pull apache/skywalking-python-agent:latest-plugin
```
The above command always runs into trouble with the error message below
```
Error response from daemon: pull access denied for apache/skywalking-python-agent, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```
Seems the latest tag is unavailable, and docker-compose.base.yml under the tests/plugin folder needs to be altered for the test in my situation.

Reply:
> > Is there any other error reported in Actions? I want to eliminate as many mistakes as possible on each commit. Thanks!
> 
> not sure, but seems like you are almost there, remember to regenerate docs and add a changelog

bear in mind

Reply:
> > It is better you could run the tests locally. Then you could catch all errors.
> 
> Yea, I make it now. Here is the problem I encountered.
> 
> ```
> docker pull apache/skywalking-python-agent:latest-plugin
> ```
> 
> The above command always runs into trouble with the error message below
> 
> ```
> Error response from daemon: pull access denied for apache/skywalking-python-agent, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
> ```
> 
> Seems the latest tag is unavailable, and docker-compose.base.yml under the tests/plugin folder needs to be altered for the test in my situation.

Sorry for my belated response. This Docker image is built on the fly right before running the tests. Please refer to https://github.com/apache/skywalking-python/blob/88b30ae094079167409e7c2bf4ff68431bc9e96a/.github/workflows/CI.yaml#L135

Reply:
> Is there any other error reported in Actions? I want to eliminate as many mistakes as possible on each commit. Thanks!

Hi. Can you see the actions log [here](https://github.com/apache/skywalking-python/runs/7165080924?check_suite_focus=true#step:6:177)? There is a diff output that indicates what’s the expected data and what’s the actual data (ignore the matching pattern like `gt 0`, `not null`, etc.)